src/mono/wasm/testassets/BlazorHostedApp/BlazorHosted.Client/Program.cs

```diff
@@ -9,5 +9,6 @@
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+builder.Logging.SetMinimumLevel(LogLevel.Debug);
```

Comment:
What is the motivation here? Typically logging is controlled by `appsettings.json` https://github.com/dotnet/runtime/blob/main/src/mono/wasm/testassets/BlazorHostedApp/BlazorHosted.Server/appsettings.json#L5

Reply:
Motivation: rn when we fail, we have a log like this [log](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-100074-merge-ce455e8c684c452684/Workloads-NoWebcil-Wasm.Build.Tests.TestAppScenarios.SignalRClientTests/1/console.be50248e.log?helixlogtype=result) 
```
[endGroup] console.groupEnd
[endGroup] console.groupEnd
[info] Debugging hotkey: Shift+Alt+D (when application has focus)
   Wasm.Build.Tests: [Long Running Test] 'Wasm.Build.Tests.TestAppScenarios.SignalRClientTests.SignalRPassMessages', Elapsed: 00:03:59
   Wasm.Build.Tests: [Long Running Test] 'Wasm.Build.Tests.TestAppScenarios.SignalRClientTests.SignalRPassMessages', Elapsed: 00:05:59
```
When we have logs on, then we see that it's actually
```
[debug] dbug: Microsoft.AspNetCore.Components.RenderTree.Renderer[3]
      Rendering component 6 of type BlazorHosted.Client.Pages.Chat
[ERROR] PageError: RuntimeError: memory access out of bounds
    at http://localhost:5000/_framework/dotnet.native.wasm:wasm-function[8788]:0x21ef65
    at http://localhost:5000/_framework/dotnet.runtime.9.0.0-ci.ooy53j55dw.js:3:43188
    at v (http://localhost:5000/_framework/dotnet.runtime.9.0.0-ci.ooy53j55dw.js:3:222380)
    at http://localhost:5000/_framework/dotnet.runtime.9.0.0-ci.ooy53j55dw.js:3:223529
    at http://localhost:5000/_framework/dotnet.runtime.9.0.0-ci.ooy53j55dw.js:3:44879
    at callUserCallback (http://localhost:5000/_framework/dotnet.native.9.0.0-ci.6ezff52hgm.js:8:102608)
    at http://localhost:5000/_framework/dotnet.native.9.0.0-ci.6ezff52hgm.js:8:111163
```
that can be caught by the Known Issue of https://github.com/dotnet/runtime/issues/96546

Reply:
Aha, I see: appsettings.json has higher priority and we have , then this line does not make sense.